wiki/People/Beatmap_Nominators/en.md

```diff
@@ -8,137 +8,153 @@
 
-### What Qualifies as a Mod Post
-+ Upon signing up you will be asked to provide the total amount of Kudosu as well as the amount of mod posts within the last 3 months, sorted by month upon signing up. So if the announcement is posted in April the mod posts for January, February and March will be relevant. The relevant months will also be listed on the submission form.
-+ A mod post qualifies as one if it has received a Kudosu and has been created within the time span listed on the submission form.
-+ The gamemode (osu!, osu!mania, osu!catch, osu!taiko) that you modded beatmaps in does not matter as your activity in the modding community as a whole is being measured and not the activity in a certain game mode.
-+ The point in time at which the Kudosu has been awarded to you mod post does not matter, the only relevant information about a mod post is its post date and time in UTC+0 as well as the fact that it has been awarded Kudosu.
+### What is mod and kudosu score, and how is the minimum threshold calculated?
 
-### How the Required Minimum Score is Calculated
-+ This depends on the total `Kudosu Score` and the total `Mod Score` an applicant achieved over the last months. A mathematical formula is used to determine these values.
-+ `Kudosu Score` is determined according to the following formula:
+In order to determine their minimum required score, an applicant needs to summarise both their kudosu score and their mod for the three month period that encompasses each application window.
+
+A mathematical formula is used to determine both kudosu and mod score.
+
+Kudosu score is determined according to the following formula:
 
 ![Kudosu Score Formula](/wiki/shared/KDSCORE.png "Kudosu Score Formula")
 
-+ `Mod Score` per month is determined according to the following formula:
+Mod score per month is determined according to the following formula:
 
 ![Mod Score Formula](/wiki/shared/MODSCORE.png "Mod Score Formula")
 
-+ In order to determine their `Required Minimum Score`, an applicant needs to summarise their `Kudosu Score` and their `Mod Score` for month one, two and three.
 + The overall minimum threshold is set to the following values:
   + osu!taiko, osu!catch and osu!mania: 1050
   + osu!: 1100
-+ It is usually a good idea to keep their socre well above these numbers as they can vary slightly depending on the amount of applicants.
+
+It is usually a good idea to keep your score well above these numbers as they can vary slightly depending on the amount of applicants.
 
 Team Members
 ------------
 
-_Please note: All BNs speak English unless otherwise noted._ Link to [user group page](https://osu.ppy.sh/g/28).
+_Please note: All BNs speak English unless otherwise noted._ Link to [user group page](https://osu.ppy.sh/home/groups/28).
 
 Also note: the game modes columns listed denotes which game mode(s) the user would focus more at, but are not forcibly subject to that game mode(s).
```

Comment:
This line/note was something that I wrote a long time ago after some dispute of removing the game modes columns... I'm not sure if you want to rewrite (just pointing this out I guess)